docs/src/main/paradox/project/migration-guides.md

```diff
@@ -16,9 +16,10 @@ These migration notes are designed for users migrating from Akka 2.6 to Pekko 1.
 * Pekko packages start with "org.apache.pekko" instead of "akka" - e.g. `import org.apache.pekko.actor` instead of `import akka.actor`
 * Where class names have "Akka" in the name, the Pekko ones have "Pekko" - e.g. PekkoException instead of AkkaException
 * Config names use "pekko" prefix instead of "akka", e.g. `pekko.actor.provider` instead of `akka.actor.provider`
+* The Pekko node URLs use `pekko://` and `pekko.tcp//` instead of `akka://` and `akka.tcp//` respectively.
```

Comment:
Maybe  a table like would be better :

|before|after|
|-|-|
|akka://|pekko://|
|akka.tcp//|pekko.tcp//|

Reply:
I've tried to create a table but Paradox build is not rendering it properly. Something about trying to put a table inside a bulleted list is causing trouble.

Reply:
I reformatted the bulleted list so that the 2 URL schemes each get their own indented list entry